client/lib/mobile-app/index.js

```diff
@@ -9,3 +9,15 @@ export function isWpMobileApp() {
 	}
 	return navigator.userAgent && /wp-(android|iphone)/.test( navigator.userAgent );
 }
+
+/**
+ * Returns whether user is using a WooCommerce mobile app.
+ *
+ * @returns {boolean} Whether the user agent matches the ones used on the WooCommerce mobile apps.
+ */
+export function isWcMobileApp() {
+	if ( typeof navigator === 'undefined' ) {
+		return false;
+	}
+	return navigator.userAgent && /wc-(android|iphone)/.test( navigator.userAgent );
```

Comment:
Can you please update the useragent of the WCiOS? it is `wc-ios` (source: https://github.com/woocommerce/woocommerce-ios/blob/4fe952211777fad134084cbc19d4db229a37c8c9/Networking/Networking/Settings/UserAgent.swift#L40)

Reply:
Corrected, thanks!